src/v2/cookbook/debugging-in-vscode.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Debugging in VS Code and Chrome
+type: cookbook
+order: 6
+---
+
+Every application reaches a point where it's necessary to understand failures, small to large. In this recipe, we explore a few workflows for VS Code users, who are using Chrome to test.
+
+This recipe shows how to use the [Debugger for Chrome](https://github.com/Microsoft/VSCode-chrome-debug) extension with VS Code to debug Vue.js applications generated by the [Vue CLI](https://github.com/vuejs/vue-cli).
+
+## Prerequisites
+
+You must have Chrome and VS Code installed. Make sure to the latest version of [Debugger for Chrome](https://marketplace.visualstudio.com/items?itemName=msjsdiag.debugger-for-chrome) extension installed in VS Code.
+
+Install and create a project with the [vue-cli](https://github.com/vuejs/vue-cli) by running
+
+```
+npm install -g @vue/cli
+# or
+yarn global add @vue/cli
+
+vue create my-project
+```
```

Comment:
I'm wondering if we should just defer people to the instructions in the Vue CLI README. That way, we don't have to update recipes if the API of Vue CLI changes. Thoughts?

Reply:
Yeah, that makes a lot of sense to me.